src/crimson/os/seastore/onode_manager/staged-fltree/node.cc

```diff
@@ -573,11 +592,11 @@ bool LeafNode::is_level_tail() const {
   return impl->is_level_tail();
 }
 
-std::tuple<key_view_t, const onode_t*, layout_version_t> LeafNode::get_kv(
-    const search_position_t& pos) const {
+std::tuple<key_view_t, const onode_t*>
+LeafNode::get_kv_versioned(const search_position_t& pos) const {
```

Comment:
It means the retrieved `key_view_t` and `onode_t*` are only valid under the specific node layout version.

The version is incremented if the node layout is changed by insert/split... operations.

Reply:
I mean, since we don't provide an unversioned variant of this method, and `LeafNode` is inherently versioned, why add `versioned` postfix in the method name?

Reply:
renamed.